src/ripple/app/paths/Pathfinder.cpp

```diff
@@ -444,8 +444,7 @@ Pathfinder::computePathRanks(int maxPaths)
 static bool
 isDefaultPath(STPath const& path)
 {
-    // TODO(tom): default paths can consist of more than just an account:
-    // https://forum.ripple.com/viewtopic.php?f=2&t=8206&start=10#p57713
+    // FIXME: default paths can consist of more than just an account:
```

Comment:
For future reference, xrpl.org has some discussion of [default paths](https://xrpl.org/paths.html#default-paths) which may be useful for fixing this up.

No action needed here at this time though.